www/modules/custom/borg_mailers/borg_mailers_security/borg_mailers_security.module

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file borg_mailers_security.module
+ * Send out Security Anouncement emails to users who have subscribed.
+ *
+ * Users can check a box upon editing there user profile (/user when logged in)
+ * to opt into the Backdrop Security email list.
+ *
+ * Security emails will go out on publishing a security release node of Backdrop
+ * core or any Backdrop Contrib project.
+ */
+
+/**
+ * Implements hook_node_submit().
+ *
+ * @see _borg_mailers_record_mail_sent().
+ * @see _borg_mailers_security_subscribers().
+ * @see _borg_mailers_record_mail_sent().
+ */
+function borg_mailers_security_node_submit($node) {
+  // Save the node right quick so we can get an nid.
+  $node->save();
+
+  // Check if we have sent mail for this node already.
+  $sent = _borg_mailers_check_if_already_sent($node->nid);
+
+  if (!$sent) {
+    // Check for node type is a project_release or core_release and status = 1.
+    if (($node->type == 'sa') && $node->status == 1) {
+      $language = language_default();
+      $params = array('node' => $node);
+      $to = _borg_mailers_security_subscribers();
+      foreach ($to as $t) {
```

Comment:
Can we use a better variable name than `$t`? What is this, an email address? How about `$email` or `$address`?